src/bigquery.ts

```diff
@@ -80,15 +80,18 @@ export type PagedRequest<P> = P & {
   maxApiCalls?: number;
 };
 
+export type QueryResultsResponse = bigquery.IGetQueryResultsResponse &
+  bigquery.IQueryResponse;
+
 export type QueryRowsResponse = PagedResponse<
   RowMetadata,
   Query,
-  bigquery.IGetQueryResultsResponse
+  QueryResultsResponse
 >;
 export type QueryRowsCallback = PagedCallback<
   RowMetadata,
   Query,
-  bigquery.IGetQueryResultsResponse
+  QueryResultsResponse
```

Comment:
ooo this is such a good question. I thought about it for a bit and I think you're right - it is backwards compatible - I think it does merit a minor release and not just a patch release though because you are adding functionality - you're just doing so in a way that is backwards compatible.